pkg/sqlite/load_test.go

```diff
@@ -232,3 +245,162 @@ func newBundle(t *testing.T, name, pkgName string, channels []string, objs ...*u
 
 	return bundle
 }
+
+func TestGetTailFromBundle(t *testing.T) {
+	type fields struct {
+		bundles []*registry.Bundle
+		pkgs    []registry.PackageManifest
+	}
+	type args struct {
+		bundle string
+	}
+	type expected struct {
+		err  error
+		tail []string
+	}
+	tests := []struct {
+		description string
+		fields      fields
+		args        args
+		expected    expected
+	}{
+		{
+			description: "GetTailFromBundle/RemoveDefaultChannelForbidden",
+			fields: fields{
+				bundles: []*registry.Bundle{
+					newBundle(t, "csv-a", "pkg-0", []string{"alpha"}, newUnstructuredCSV(t, "csv-a", "csv-b")),
+					newBundle(t, "csv-b", "pkg-0", []string{"alpha", "stable"}, newUnstructuredCSV(t, "csv-b", "csv-c")),
+					newBundle(t, "csv-c", "pkg-0", []string{"alpha", "stable"}, newUnstructuredCSV(t, "csv-c", "")),
+				},
+				pkgs: []registry.PackageManifest{
+					{
+						PackageName: "pkg-0",
+						Channels: []registry.PackageChannel{
+							{
+								Name:           "alpha",
+								CurrentCSVName: "csv-a",
+							},
+							{
+								Name:           "stable",
+								CurrentCSVName: "csv-b",
+							},
+						},
+						DefaultChannelName: "stable",
+					},
+				},
+			},
+			args: args{
+				bundle: "csv-a",
+			},
+			expected: expected{
+				err:  registry.ErrRemovingDefaultChannelDuringDeprecation,
+				tail: nil,
+			},
+		},
+		{
+			description: "GetTailFromBundle/RemovingNonDefaultChannel",
+			fields: fields{
+				bundles: []*registry.Bundle{
+					newBundle(t, "csv-a", "pkg-0", []string{"alpha"}, newUnstructuredCSV(t, "csv-a", "csv-b")),
+					newBundle(t, "csv-b", "pkg-0", []string{"alpha", "stable"}, newUnstructuredCSV(t, "csv-b", "csv-c")),
+					newBundle(t, "csv-c", "pkg-0", []string{"alpha", "stable"}, newUnstructuredCSV(t, "csv-c", "")),
+				},
+				pkgs: []registry.PackageManifest{
+					{
+						PackageName: "pkg-0",
+						Channels: []registry.PackageChannel{
+							{
+								Name:           "alpha",
+								CurrentCSVName: "csv-a",
+							},
+							{
+								Name:           "stable",
+								CurrentCSVName: "csv-b",
+							},
+						},
+						DefaultChannelName: "alpha",
+					},
+				},
+			},
+			args: args{
+				bundle: "csv-a",
+			},
+			expected: expected{
+				err: nil,
+				tail: []string{
+					"csv-b",
+					"csv-c",
+				},
+			},
+		},
+		{
+			description: "GetTailFromBundle/HandlesSkips",
+			fields: fields{
+				bundles: []*registry.Bundle{
+					newBundle(t, "csv-a", "pkg-0", []string{"alpha"}, newUnstructuredCSVwithSkips(t, "csv-a", "csv-b", "\"csv-d\",\"csv-e\",\"csv-f\"")),
```

Comment:
What does `"\"csv-d\",` mean? Does this represent skips?

Reply:
"\"csv-d\",\"csv-e\",\"csv-f\"" is a list of skipped csvs